### PR TITLE
[incubator/logstash] Add README file and change service type

### PR DESCRIPTION
**Pull request by christian-roggia**
# TL;DR
This PR adds the required README to the chart in order to provide basic documentation. Some information about the chart has also been updated.

# ClusterIP vs NodePort
Since most user won't send requests to Logstash from outside the cluster the default service type has been changed from NodePort to ClusterIP. This decision has been made since we assume that all the traffic will be generated from within the cluster, the same decision has been applied to most charts available in this repository and also to the ElasticSearch chart itself. Also, exposing the logstash service by default with NodePort is a security issue.

### CODE REVIEW

**Comment by rendhalver** [comment]
Sounds good to me. :)
